lib/ci-stack.ts

```diff
@@ -80,6 +90,33 @@ export class CIStack extends Stack {
       oidcCredArn: importedOidcConfigValuesSecretBucketValue.toString(),
       useSsl,
       runWithOidc,
+    },
+    {
+      region: this.region.toString(),
+      agent_node_security_group: securityGroups.agentNodeSG.securityGroupId.toString(),
+      subnet_id: vpc.publicSubnets[0].subnetId.toString(),
+    },
+    {
```

Comment:
Lets move these into the `AgentNodeConfig` file, otherwise this `Stack` contains a whole lot of little detail that makes it hard to understand the pieces that are going into the stack.  It could look something like:

```
export class AgentNodeConfig {
   static readonly AGENT_CONFIGS: CloudAgentConfig[] = [{
      ec2_cloud_name: 'AL2-X64',
      ...
   }]
}
```

This way when we add update the configuration only one clearly organized place is updated.

Reply:
Done